integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -185,19 +193,50 @@ func ConfirmExecWithSeqNr(
 			if err != nil {
 				return fmt.Errorf("error to get source chain config : %w", err)
 			}
-			t.Logf("Waiting for ExecutionStateChanged on chain  %d from chain %d with expected sequence number %d, current onchain minSeqNr: %d",
-				dest.Selector, source.Selector, expectedSeqNr, scc.MinSeqNr)
+			executionState, err := offRamp.GetExecutionState(nil, source.Selector, expectedSeqNr)
+			if err != nil {
+				return fmt.Errorf("error to get execution state : %w", err)
+			}
+			t.Logf("Waiting for ExecutionStateChanged on chain %d (offramp %s) from chain %d with expected sequence number %d, current onchain minSeqNr: %d, execution state: %s",
+				dest.Selector, offRamp.Address().String(), source.Selector, expectedSeqNr, scc.MinSeqNr, executionStateToString(executionState))
+			if executionState == EXECUTION_STATE_SUCCESS {
+				t.Logf("Observed SUCCESS execution state on chain %d (offramp %s) from chain %d with expected sequence number %d",
+					dest.Selector, offRamp.Address().String(), source.Selector, expectedSeqNr)
+				return nil
+			}
```

Comment:
Adding this because in the flakes I observed the following phenomenon:

```
2024-09-18T16:35:18.1257108Z     Waiting for ExecutionStateChanged on chain 5548718428018410741 (offramp 0x5E6f8839581AfeE68f899bF33B8F284FFf92ef0E) from chain 909606746561742123 with expected sequence number 1, current onchain minSeqNr: 2, execution state: UNTOUCHED
2024-09-18T16:35:18.1258349Z     Waiting for ExecutionStateChanged on chain 909606746561742123 (offramp 0xBAa5D0009A5a445ab05F190923dBFd0624BC9dd7) from chain 5548718428018410741 with expected sequence number 1, current onchain minSeqNr: 2, execution state: UNTOUCHED
2024-09-18T16:35:18.1259586Z     Waiting for ExecutionStateChanged on chain 5548718428018410741 (offramp 0x5E6f8839581AfeE68f899bF33B8F284FFf92ef0E) from chain 789068866484373046 with expected sequence number 1, current onchain minSeqNr: 2, execution state: SUCCESS
2024-09-18T16:35:18.1260817Z     Waiting for ExecutionStateChanged on chain 909606746561742123 (offramp 0xBAa5D0009A5a445ab05F190923dBFd0624BC9dd7) from chain 789068866484373046 with expected sequence number 1, current onchain minSeqNr: 2, execution state: SUCCESS
2024-09-18T16:35:18.1262055Z     Waiting for ExecutionStateChanged on chain 789068866484373046 (offramp 0xAF84A0E10C914dD6d91c2e1674031f4668Ada3FB) from chain 909606746561742123 with expected sequence number 1, current onchain minSeqNr: 2, execution state: UNTOUCHED
2024-09-18T16:35:18.1263291Z     Waiting for ExecutionStateChanged on chain 789068866484373046 (offramp 0xAF84A0E10C914dD6d91c2e1674031f4668Ada3FB) from chain 5548718428018410741 with expected sequence number 1, current onchain minSeqNr: 2, execution state: UNTOUCHED
```

i.e even before subscribing it seems that we successfully executed some messages. The subscription never ended up picking up these logs and relaying them to the channel. So this is hopefully going to aid in those cases.